### DISCUSSION TITLE
Aderyn report problem

### DISCUSSION THREAD

**Discussion by exolorkistis** [Cyfrin/security-and-auditing-full-course-s23#94] [Q&A]
I am using the aderyn command : aderyn . in my project folder and while i dont get any errors and the report.md is created it does not show any  of the  results it shows on the tutorials?Is there something i can do?. I found that there are similar discussions but there is no answer there so i thought i would create  a new one.

**Comment by johnsonstephan** [comment]
Hi @exolorkistis, thanks for surfacing the issue. We will require more context to assist you with your specific issue. [See this](https://github.com/Cyfrin/security-and-auditing-full-course-s23/blob/main/how-to-ask-a-question.md) on asking a question.

**Reply by exolorkistis** [reply]
This question has been asked before but there wasnt any answer that could help me.I am using VSCode and i am using Gitbash terminal.So i installed aderyn according to the repository guidance and while it runs without errors when i give the ```aderyn .``` command and the report.md is created the report never finds any results even in the projects of the course.

Below is the text i get when running the command:

```
$ aderyn .
Detecting framework...
Framework detected: Foundry mode engaged.
Foundry root path: "."
[⠒] Compiling...
[⠆] Compiling 41 files with 0.8.20
[⠔] Solc 0.8.20 finished in 15.36s
Compiler run successful!
Foundry src path: ".\\src"
Get Detectors
Running 18 detectors
Detectors run, processing found issues 
Found issues processed. Printing report
Report printed to report.md
```

A report.md is created but everytime it looks like this with no results:

```
# Aderyn Analysis Report

This report was generated by [Aderyn](https://github.com/Cyfrin/aderyn), a static analysis tool built by [Cyfrin](https://cyfrin.io), a blockchain security company. This report is not a substitute for manual audit or security review. It should not be relied upon for any purpose other than to assist in the identification of potential security vulnerabilities.
# Table of Contents

- [Summary](#summary)
  - [Files Summary](#files-summary)
  - [Files Details](#files-details)
  - [Issue Summary](#issue-summary)


# Summary

## Files Summary

| Key | Value |
| --- | --- |
| .sol Files | 0 |
| Total nSLOC | 0 |


## Files Details

| Filepath | nSLOC |
| --- | --- |
| **Total** | **0** |


## Issue Summary

| Category | No. of Issues |
| --- | --- |
| Critical | 0 |
| High | 0 |
| Medium | 0 |
| Low | 0 |
| NC | 0 |



```

How can i make it work?Is there something else i should installed that it is not mentioned in the repository?

**Reply by PatrickAlphaC** [reply]
Could make this an issue on the aderyn GitHub with the exact codebase you're getting no results on? 

Aderyn git: https://github.com/Cyfrin/aderyn
